text/0000-rfc-process-update.md

```diff
@@ -0,0 +1,166 @@
+
+### Merge ember-cli/rfcs into emberjs/rfcs
+
+Part of removing confusion about the process is having a central repository for all Ember RFCs.
+
+To achieve merging ember-cli/rfcs into emberjs/rfcs the following will be done:
+
+* Remove active/completed concepts from ember-cli/rfcs
+  * Add front matter
+  * Copy both active and completed RFC files into `text` of emberjs/rfcs
+* Transfer active PRs and Issues to emberjs/rfcs
+* Archive ember-cli/rfcs
+
+There are some concerns about links breaking when we move the files to emberjs/rfcs,
+but given the fact that ember-cli/rfcs had the concept of active/completed by moving the files into different folders,
+links were already being broken.
+
+The ember-cli/rfcs do not need name or numbering changes, as there is currently no duplicated name.
+Going forward, the numbering should be unified by virtue of having a single repository.
+
+### Open process to more types of RFCs
+
+To build on the previous point, the front matter that is to be added to RFCs will make it easier to have RFCs for other areas of the project,
+like the design of the web resources, Guides content, marketing content, or features for core addons.
+
+Given that Ember is [organized into teams](https://emberjs.com/team/), with each team being responsible for certain projects,
+the RFC front matter should be oriented towards those teams.
+
+A list of the teams and respective projects should be added to the instructions,
+possibly with the addition of per-team instructions on specifics of the project.
+Additional templates might be created as well, such a design work template.
+
+### Provide more guidance for post-RFC
+
+At the moment it is not clear for some contributors what happens once an RFC is merged.
+How and when should work to implement an RFC be done?
+
+## How We Teach This
+
+To ensure that contributors are updated on the RFC process and the process is clear,
+the documentation should be improved in a couple of ways.
+
+The README should be updated to include information about Champions,
+how they are assigned, what they do.
+The README should include a description of the lifecycle of RFCs with the new "FCP to close" state,
+and instructions on how to proceed with the implementation of the RFC once it is accepted.
+
+## Drawbacks
+
+### Adjustment period
+
+There are active RFCs in ember-cli/rfcs. Moving these discussions would be onerous, so they should be kept there until completion, and no new RFCs accepted.
+
+### Permalinks to ember-cli/rfcs proposals
+
+Moving the RFC files from ember-cli/rfcs (active or completed) to emberjs/rfcs can be seen as a breaking change, and could lead to someone linking to ember-cli/rfcs and then the RFC being updated in emberjs/rfcs. However, ember-cli/rfcs already suffers from a linking problem due to the active/completed folders, as RFCs need to be moved from one to the other even after being accepted.
```

Comment:
Repetition with "There are some concerns about links breaking..." I recommend removing the "There are concerns" section and working it into here